app/Http/Controllers/Areas/AreasController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers\Areas;
+
+use App\Models\DBC\Area\AreasAndZones as AreasAndZonesWotlk;
+use App\Models\WOD\DBC\Area\AreasAndZones as AreasAndZonesWod;
+use App\Exceptions\UnsupportedVersion;
+use App\Helpers\TCAPI;
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use Illuminate\Routing\Controller;
+
+class AreasController extends Controller
+{
+    private $api;
+
+    /**
+     * AchievementsController constructor.
+     *
+     * @param TCAPI $api
+     */
+    public function __construct(TCAPI $api)
```

Comment:
https://github.com/ShinDarth/TC-JSON-API/commit/cef00b36efffa99afb0b82f6f51964bfee5ad12a#diff-80bda0ca3af455d6354e095203e0199aR25
For what I need write more code? If that already do other people?